relayer/crates/starknet-chain-components/src/impls/queries/packet_receipt.rs

```diff
@@ -112,6 +112,6 @@ where
             proof_bytes: signed_bytes,
         };
 
-        Ok((vec![], dummy_proof))
+        Ok((Some(vec![]), dummy_proof))
```

Comment:
shouldn't this be `None`?

Reply:
This has to be `None`. Otherwise the non-membership verification on wasm light client will fail.

https://github.com/informalsystems/ibc-starknet/blob/3a2029e468a86042bdb9afda0eb5c44aa590fbef/relayer/crates/starknet-chain-components/src/impls/queries/packet_receipt.rs#L101-L102

Reply:
44c19f4